templates/credentials/base_cred.html

```diff
@@ -9,6 +9,57 @@
 {% endblock body_class %}
 
 {% block outer_content %}
+  {% if show_cred_maintenance_alert and cred_maintenance_start and cred_maintenance_end %}
+    <div class="p-notification--information" style="margin:1rem">
+      <div class="p-notification__content">
+        <h5 class="p-notification__title">Maintenance Scheduled</h5>
+        <p class="p-notification__message" id="cred-maintenance-message"></p>
+      </div>
+    </div>
+    <script>
+      const formatter = (dateTime, locale = 'default') => {
```

Comment:
It would help if the variable names clearly showed what they’re for instead of just being called `formatter`